drill-admin/src/main/kotlin/com/epam/drill/endpoints/ServerWsTopics.kt

```diff
@@ -89,6 +89,10 @@ class ServerWsTopics(override val kodein: Kodein) : KodeinAware {
                         pluginWebSocket
                     }
                 }
+
+                topic<WsRoutes.GetPluginConfig> { ids, _ ->
+                    agentManager[ids.agentId]?.plugins?.find { it.id == ids.pluginId }?.config
```

Comment:
ids? what is it?  agentId, pluginId??? )))

Reply:
this is for agent's and plugin's ids

Reply:
is it better to call it some 'x', like at row 68?